base-notebook/docker_healthcheck.py

```diff
@@ -11,7 +11,7 @@
 # As this is a healthcheck, it should succeed or raise an exception on error
 
 runtime_dir = Path("/home/") / os.environ["NB_USER"] / ".local/share/jupyter/runtime/"
-json_file = next(runtime_dir.glob("*.json"))
+json_file = next(runtime_dir.glob("jpserver*.json"))
```

Comment:
```suggestion
json_file = next(runtime_dir.glob("jpserver-*.json"))
```

Let's add `-` to be even more specific.